src/server/scripts/Events/hallows_end.cpp

```diff
@@ -1419,6 +1435,33 @@ class go_loosely_turned_soil : public GameObjectScript
     }
 };
 
+class go_pumpkin_shrine : public GameObjectScript
+{
+public:
+    go_pumpkin_shrine() : GameObjectScript("go_pumpkin_shrine") {}
+
+    bool OnGossipSelect(Player* player, GameObject* go, uint32 /*sender*/, uint32 /*action*/) override
+    {
+        CloseGossipMenuFor(player);
```

Comment:
According to the CI Workflows, this is not even being declared and i believe it can only be used for NPC gossips.
Therefore it gets skips over
```suggestion
        //CloseGossipMenuFor(player);
```